react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+export default function ToggleSwitch() {
+
+  const [off, on] = useState(false);
```

Comment:
These names are a bit misleading since you can have `off` be `true` and `on` turn the switch off. Something like `isOn` or `isSelected` would be more clear. Then you would use names that match: `const [isOn, setIsOn] = useState(false);`